Fig4_PyGMT_pandas.py

```diff
@@ -0,0 +1,64 @@
+import io
+
+import pygmt
+import pandas as pd
+import requests
+
+url = "https://earthquake.usgs.gov/fdsnws/event/1/query"
+params = {
+    "format": "csv",
+    "starttime": "2000-01-01",
+    "endtime": "2025-10-30",
+    "mindepth": 70,
+    "minlongitude": 91,
+    "maxlongitude": 134,
+    "minlatitude": -21,
+    "maxlatitude": 19,
+    "minmagnitude": 5,
+    "orderby": "magnitude",
+}
+r = requests.get(url, params=params)
+df_eqs = pd.read_csv(io.StringIO(r.text))
+df_eqs = df_eqs[df_eqs["magType"] != "mb"]  # Focus on moment magnitudes
+
+fig = pygmt.Figure()
+fig.basemap(region=[91, 134, -21, 19], projection="M15c", frame=True)
+fig.coast(land="gray95", shorelines="gray50")
+
+# Plot epicenters with color (hypocentral depth) or size (moment magnitude)
+pygmt.makecpt(cmap="SCM/navia", series=[70, 700], reverse=True, transparency=30)

```

Comment:
What about making the CPT range to 0-700?
```suggestion
pygmt.makecpt(cmap="SCM/navia", series=[0, 700], reverse=True, transparency=30)
```